### ISSUE TITLE
Please tag releases with git tags

### ISSUE THREAD

**Issue by alanorth** [isovic/graphmap#99]
The README.md says the latest version is v0.5.2, but I don't see a git tag for that version. It would make it easier for those of us installing graphmap into clustered environments if the source code had proper tags. Thanks!

**Comment by orbsmiv** [comment]
Yes, I second this as I'm currently having to version the application with its commit hash instead of version number to ensure future reproducibility. (We run a module system for our HPC applications.)
